src/web/core/src/main/java/org/geoserver/web/css/geoserver.css

```diff
@@ -925,6 +925,33 @@ padding-top: 18px;
 .page-footer {
 padding-top: 1em;
 }
+/* action group above tables */
+.action-group li {
+  display: inline;
+}
+
+/* internationalization */
+.i18n-panel {
+  margin-bottom: 5px;
+}
+
+.i18n-panel td {
+  vertical-align: top;
+}
+
+/*
+.i18n-text .text {
+  width: 80%;
+}
+
+.i18n-tile-and-abstract .text {
+  width: 100%;
+}
+
+.i18n-panel .text {
+  width: 100%;
+}
+*/
```

Comment:
The above is commented out, could not manage to get the i18n table to have a sensible layout.

Reply:
Commented out bits are not welcomed in PRs though?
Unfortunately we don't have any CSS expert in our server-side team...